sdk/identity/azure-identity/test/ut/token_credential_test.cpp

```diff
@@ -55,6 +55,11 @@ using namespace Azure::Identity;
 
 TEST_F(TokenCredentialTest, ClientSecret)
 {
+  if (GetEnv("AZURE_TEST_MODE") == "LIVE" || GetEnv("AZURE_TEST_MODE") == "live")
```

Comment:
Call GetEnv once to get the value and then compare that value.

string envVar = GetEnv("AZURE_TEST_MODE");
if (envVar == "LIVE" || envVar == "live)

Reply:
Or even better would be to put this logic into a test helper function then do `if(IsLiveTestMode())`